Description: Fix TAB behaviour in prompt
 Sadly the 'no echo' semi-hidden feature would require TAB+backspace since the
 TAB is still registered as a password character.
Author: Marc Schoolderman <<REDACTED_EMAIL>>
---
Origin: upstream, commit: 483b9a5
Forwarded: not-needed
Applied-Upstream: commit: 483b9a5
Reviewed-By: Simon Johnsson <<REDACTED_EMAIL>>
Last-Update: 2026-03-11

--- a/src/pam/rpassword.rs
+++ b/src/pam/rpassword.rs
@@ -263,6 +263,7 @@ fn read_unbuffered(
             if read_byte == b'\t' && feedback.visible_len.take().is_some() {
                 feedback.clear();
                 let _ = feedback.sink.write(b"(no echo)");
+                continue;
             }
         }

